pkg/util/jsonpath/jsonpath_test.go

```diff
@@ -49,6 +52,31 @@ func testJSONPath(tests []jsonpathTest, t *testing.T) {
 	}
 }
 
+// testRandomReulst test testcases related to map, the results may print in random order
```

Comment:
I would suggest calling this testJSONPathSortOutput, to explicitly convey that it sorts output before comparing with expected.